exo-deploy/terraform/aws/aws.tf

```diff
@@ -0,0 +1,60 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_availability_zones" "available" {}
+
+module "internal_dns" {
+  source = "./internal-dns"
+
+  name    = "${var.name}.local"
```

Comment:
I think I would want the local name to be consistent regardless of the environment. For example `users.myapp.local` should just go to the users service within the correct environment. I wouldn't want to have to do `users.prod-myapp.local` for one environment and `users.dev-myapp.local` for another when the two can't even conflict